assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -90,7 +98,8 @@ class Draggable extends Component {
 			// All this is about calculating the position of the (correct) element on the new page.
 			const currentElementTop = parseInt( this.cloneWrapper.style.top );
 			const currentElementLeft = parseInt( this.cloneWrapper.style.left );
-			const newLeft = currentElementLeft - ( this.pageOffset * PAGE_AND_MARGIN );
+			const factor = isRTL ? 1 : -1;
+			const newLeft = currentElementLeft + factor * ( this.pageOffset * PAGE_AND_MARGIN );
```

Comment:
> `Unexpected mix of '+' and '*'  no-mixed-operators`

Reply:
```suggestion
			const newLeft = currentElementLeft + ( factor * ( this.pageOffset * PAGE_AND_MARGIN ) );
```

Reply:
*Please Excuse My Dear Aunt Sally*

(also, fixing 😜 )

Reply:
Fixed in a11831f.